config/default.yaml

```diff
@@ -34,6 +34,12 @@ builds:
     # Upper limit on incoming uploads to builds
     maxByteSize: 1073741824 # 1GB
 
+caches:
+    # How long should caches stay around for
+    expiresInSec: 2592000000 # 30 days
```

Comment:
make it also configurable in here? https://github.com/screwdriver-cd/store/blob/master/config/custom-environment-variables.yaml

Reply:
added